vitrina/datasets/models.py

```diff
@@ -977,7 +986,79 @@ def get_metadata_objects_for_version(self):
+            metadata_model = metadata_instance.content_type.model
+            if metadata_model == "prefix":
+                label = mark_safe(
+                    f"Prefix ref: "
+                    f"<span class='tag is-success is-light is-medium prop_metadata'>{metadata_instance.name}</span>"
+                )
+
+                meta_objects.append((metadata_instance.pk, label))
+
+            if metadata_model == "paramitem":
+                param_item_instance = (
+                    ParamItem.objects.filter(metadata=metadata_instance).select_related("param").first()
+                )
+                if param_item_instance:
+                    if param := param_item_instance.param:
+                        if (
+                            param.content_type_id == dataset_content_type_pk
+                            and metadata_instance.metadata_version.status == VersionStatus.DRAFT
+                        ):
+                            if metadata_instance.ref:
+                                dataset_param_item_metadata_ref = metadata_instance.ref
+                            label = mark_safe(
+                                f"Param ref:  <span class='tag is-success is-light is-medium prop_metadata'>{dataset_param_item_metadata_ref}</span>"
+                                f" prepare: <span class='tag is-success is-light is-medium'>{metadata_instance.prepare}</span>"
+                            )
+                            meta_objects.append((metadata_instance.pk, label))
+
+            if metadata_model == "enumitem":
+                enum_item_instance = EnumItem.objects.filter(metadata=metadata_instance).select_related("enum").first()
+                if enum_item_instance:
+                    if enum := enum_item_instance.enum:
+                        if (
+                            enum.content_type_id == dataset_content_type_pk
+                            and metadata_instance.metadata_version.status == VersionStatus.DRAFT
+                        ):
+                            if metadata_instance.ref:
+                                dataset_enum_item_metadata_ref = metadata_instance.ref
+                            label = mark_safe(
+                                f"Enum ref:  <span class='tag is-success is-light is-medium prop_metadata'>{dataset_enum_item_metadata_ref}</span> "
+                                f" prepare: <span class='tag is-success is-light is-medium'>{metadata_instance.prepare}</span>"
+                            )
+                            meta_objects.append((metadata_instance.pk, label))
+
         for model in self.model_set.all():
+            dataset_distribution_for_model = model.distribution
+            dataset_distribution_metadata = (
+                dataset_distribution_for_model.metadata.first()
+                if dataset_distribution_for_model and dataset_distribution_for_model.metadata.exists()
+                else None
+            )
```

Comment:
```suggestion
            dataset_distribution_metadata = (
                dataset_distribution_for_model.metadata.first() if dataset_distribution_for_model else None
            )
```
No point to check if exists(), first() will return None if nothing found.

Reply:
Thanks!